snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -149,7 +149,40 @@ func (b *bootstrapper) Start(ctx context.Context, startReqID uint32) error {
 	b.startingHeight = lastAccepted.Height()
 	b.Config.SharedCfg.RequestID = startReqID
 
-	if !b.StartupTracker.ShouldStart() {
+	return b.startBootstrappingOnceIfSufficientlyConnected(ctx)
```

Comment:
`startBootstrappingOnceIfSufficientlyConnected` is pretty clear but I think their are not really idomatic in go. How about we rename to something like "tryStart" and we explain in a comment we're looking for enough connected stake?

Reply:
ok